chaos/history.py

```diff
@@ -0,0 +1,38 @@
+
+import json
+
+from flask_restful import marshal
+from flask import g
+from fields import disruption_fields
+from chaos.models import HistoryDisruption
+from chaos import db
+
+def save_in_database(disruption_id, disruption_json):
+    history_disruption = HistoryDisruption()
+
+    history_disruption.disruption_id = disruption_id
+    history_disruption.data = disruption_json
+    db.session.add(history_disruption)
+    db.session.commit()
+
+def clean_before_save_in_history(disruption):
+    for key in disruption.keys():
```

Comment:
To fix error when we have multiple channel

```suggestion
    for key in disruption.keys():
        if isinstance(disruption, str):
        return
```